cobra/config.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2014 Novo Nordisk Foundation Center for Biosustainability, DTU.
```

Comment:
Apache and Gpl are not compatible. We would have to change the license to GPL/LGPL here.